internal/app/wwctl/profile/set/main.go

```diff
@@ -70,11 +70,6 @@ func CobraRunE(cmd *cobra.Command, args []string) error {
 			p.Root.Set(SetRoot)
 		}
 
-		if SetAssetKey != "" {
-			wwlog.Printf(wwlog.VERBOSE, "Profile: %s, Setting asset key to: %s\n", p.Id.Get(), SetAssetKey)
-			p.AssetKey.Set(SetAssetKey)
-		}
-
```

Comment:
While I wouldn't personally recommend this, we don't have to force asset keys to be unique, so let's allow this in a profile configuration.